portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
+#barTop{
+    margin-bottom: 15px;
```

Comment:
A lot of your spacing looks like four spaces instead of two to me, so I don't think what you tried doing with Cloud Shell worked :(

Reply:
Done! I realized my mistake with the Cloud Shell configuration but it is working now!